.github/workflows/test.yml

```diff
@@ -27,6 +32,7 @@ jobs:
       with:
         ruby-version: ${{ matrix.ruby_version }}
         bundler-cache: true
+    - name: Install dependencies
+      run: bundle install
```

Comment:
this is not needed since `bundler-cache: true` on previous step

Reply:
👍